package/health_daemon/src/baseline_monitor.c

```diff
@@ -86,29 +90,33 @@ static int sbp_msg_baseline_ecef_callback(health_monitor_t* monitor, u16 sender_
   return result;
 }
 
-static int baseline_threshold_rate_limiting_timer_callback(health_monitor_t* monitor, void *context)
+static int baseline_threshold_rate_limiting_timer_callback(health_monitor_t *monitor,
+                                                           void *context)
 {
   (void)context;
   log_fn_t log_fn = health_monitor_get_log(monitor);
   if (baseline_monitor_ctx.past_threshold) {
-    log_fn(LOG_WARNING, "Baseline Distance Over Threshold: %.4fm", baseline_monitor_ctx.distance_over_threshold);
+    log_fn(LOG_WARNING,
```

Comment:
Should have asked earlier, but what is the motivation for the indirection with log_fn?  Why not use piksi_log or sbp_log directly?

Reply:
I think this could go away. It was a thought to add functionality later that would insert the monitor name into the log function call since there is only a single shared [log identity](https://github.com/swift-nav/piksi_buildroot/blob/c16da4843386031d1630e5e1d02c6dd18a06dec4/package/libpiksi/libpiksi/src/logging.c#L19). But I didn't end up finishing that part of the implementation.